examples/nginx/conf/default.conf

```diff
@@ -0,0 +1,45 @@
+server {
+    listen       80;
+    listen       8080;
+    server_name  localhost;
```

Comment:
Reading [this](http://nginx.org/en/docs/http/server_names.html) I suspect we will need to set server_name to *